src/server/family_utils.cc

```diff
@@ -40,4 +47,25 @@ RandomPick UniquePicksGenerator::Generate() {
   return max_index;
 }
 
+streamConsumer* StreamCreateConsumer(streamCG* cg, string_view name, uint64_t now_ms, int flags) {
```

Comment:
moved here from t_stream because we pass now_ms (the only differrence) instead of calling directly `mstime()`.
In later versions of Redis/Valkey they also switched to "command time" by callng a global getter. Dragonfly uses transactional time